Team Review PR (#1)
MicrosoftDocs/terminal: TerminalDocs/panes.md

```

## Resizing a pane

You can adjust the size of your panes by holding `Alt+Shift` and using your arrow keys to resize the focused pane.

![Windows Terminal create pane](./images/resize-panes.gif)

To customize this key binding, you can add new ones using the `resizePane` action and setting the `direction` to either `down`, `left`, `right`, or `up`.

```json
{ "command": { "action": "resizePane", "direction": "down" }, "keys": "alt+shift+down" },
{ "command": { "action": "resizePane", "direction": "left" }, "keys": "alt+shift+left" },
{ "command": { "action": "resizePane", "direction": "right" }, "keys": "alt+shift+right" },
{ "command": { "action": "resizePane", "direction": "up" }, "keys": "alt+shift+up" }
```

## Closing a pane

You can close the focused pane by typing `Ctrl+Shift+W`. If you only have one pane, `Ctrl+Shift+W` will close the tab.

![Windows Terminal close panes](./images/close-panes.gif)

You can change what keys close the pane by adding a key binding that uses the `closePane` command.

```json
{ "command": "closePane", "keys": "ctrl+shift+w" }
```

## Customizing panes using key bindings

> You can customize the what opens inside a new pane depending on your custom key bindings.

Reviewer: ```suggestion
You can customize which profile opens inside a new pane depending on your custom key bindings.
```

Author: I'm not sure exactly how to word this, since it's not only the profile. You can do startingDirectory, commandline, etc.

### Duplicating a pane

The Terminal allows you to duplicate the focused pane's profile into another pane.

![Windows Terminal duplicate panes](./images/duplicate-panes.gif)

This can be done by adding the `splitMode` property with `duplicate` as the value to a `splitPane` key binding.

```json
{ "command": { "action": "splitPane", "split": "auto", "splitMode": "duplicate" }, "keys": "alt+shift+d" }
```

### New Terminal arguments

> [!INCLUDE [new-terminal-arguments](./new-terminal-arguments.md)]

Reviewer: Clever.
